silkworm/silkrpc/commands/rpc_api_table.cpp

```diff
@@ -22,6 +22,8 @@
 #include <silkworm/silkrpc/common/log.hpp>
 #include <silkworm/silkrpc/http/methods.hpp>
 
+#define API_WITH_GLAZE 1
```

Comment:
You can remove this switch and leave just the new implementation based on Glaze for eth_getLogs. Next PR is OK